src/index.js

```diff
@@ -27,6 +27,18 @@ const polyfillAtRuntime = ['childNodes', 'parentNode'];
 // Some properties that should not be overridden in the Text prototype.
 const doNotOverridePropertiesInTextNodes = ['textContent'];
 
+// Some new properties that should be defined in the Text prototype.
+const defineInTextNodes = ['assignedSlot'];
+
+// Some properties that should not be overridden in the Comment prototype.
+const doNotOverridePropertiesInCommNodes = ['textContent'];
+
+// Some new properties that should be defined in the Comment prototype.
+const defineInCommNodes = [];
+
+// Nodes that should be slotted
+const slottedNodeTypes = [1, 3];
```

Comment:
Might be nice to somehow reference that this is element and text nodes – maybe link to https://developer.mozilla.org/en/docs/Web/API/Node/nodeType in a comment, or better yet, using Node.ELEMENT_NODE / Node.TEXT_NODE